Source/TrackableEntities.EF.5/DbContextExtensions.cs

```diff
@@ -523,6 +525,17 @@ private static bool IsComplexType(this DbContext dbContext, Type entityType)
             return oType.BuiltInTypeKind == BuiltInTypeKind.ComplexType;
         }   
 
+        private static bool IsMappedProperty(this DbContext dbContext, Type entityType, string propertyName)
+        {
+            MetadataWorkspace workspace = ((IObjectContextAdapter)dbContext)
+                .ObjectContext.MetadataWorkspace;
+
+            StructuralType oType = workspace.GetItems<StructuralType>(DataSpace.OSpace)
+                .Where(e => e.FullName == entityType.FullName).SingleOrDefault();
+
+            return oType.Members.Select(member => member.Name).Contains(propertyName);
```

Comment:
The four lines in this method, are identical to the IsComplexType and another method, shouldn't we integrate between them?